naming/program/Cargo.toml

```
@@ -0,0 +1,29 @@
+[package]
+name = "spl-naming"
```

WDYT about calling this the:
```suggestion
name = "spl-nam-service"
```
and then moving `naming/*` into `/name-service/*`?

---

sounds good!